src/sqlparse_identifiers_match.erl

```diff
@@ -0,0 +1,161 @@
+%% -----------------------------------------------------------------------------
+%%
+%% sqlparse_identifiers_match.erl: SQL - matching the identifiers
+%%                                       of a SQL statement.
+%%
+%% Copyright (c) 2012-18 K2 Informatics GmbH.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -----------------------------------------------------------------------------
+
+-module(sqlparse_identifiers_match).
+
+-export([
+    finalize/2,
+    fold/5,
+    init/1
+]).
+
+-define(NODEBUG, true).
+
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
This makes this module looks like it only contains eunit tests. When actually there are no eunit tests in it. Why do we need this here?

Reply:
?debugFmt.